query/task_manager.go

```diff
@@ -315,6 +318,12 @@ func (t *TaskManager) waitForQuery(qid uint64, interrupt <-chan struct{}, closin
 
 		t.queryError(qid, err)
 	case <-timerCh:
+		if t.LogTimedoutQueries {
+			t.Logger.Warn(
+				"query killed for exceeding timeout limit",
+				zap.String("query", t.queries[qid].query),
```

Comment:
We could put a little more information in here, as we do in `LogCurrentQueries`: particularly the database and the elapsed time. "query killed for exceeding timeout limit of 30s on database MyDatabase"  This will save some lookup for the reader, e.g., "What was that timeout?"

Reply:
Perhaps this is out of scope, but in addition to logging the time limit / database many users want to know the user/client that executed query. eg "query sent from iclark@influxdata.com against database: Foodb exceeded timeout of 1 mins"

Reply:
Hey @idclark, while I like the suggestion unfortunately the plumbing to support that doesn't exist and would be a pretty heavy overhaul to implement. Something we can keep in mind if we are ever doing work in that area though.